resources.py

```diff
@@ -128,4 +128,30 @@ def refresh_tokens():
         token = create_token(user_id)
         return jsonify(token), 200
     except:
-        return jsonify({'error':'invalid token'}), 400
\ No newline at end of file
+        return jsonify({'error':'invalid token'}), 400
+
+@resources.route('/documents', methods=['POST'])
+def create_documents():
+    try:
+        auth_header = request.headers.get('Authorization')            
+        user_id = token_valid_check(auth_header)
+        user = get_user_by_user_id(user_id)
+        if user is None:
```

Comment:
tlhis error doesn't make sense no? cos if the user_id is invalid, the token would not have been valid; 
only exception to my statement is if the user was created, token was sent to the client (browser), and then the user was deleted from the DB - which should not be happening neways; tell me why @manthankumbhar

Reply:
true, doesn't make sense, was thinking it some other way ig :" my bad